Dumper/TranslationDumper.php

```diff
@@ -194,6 +211,41 @@ private function dumpTranslations($route, array $formats, $target)
         }
     }
 
+    private function dumpMergedTranslations($route, array $formats, $target)
+    {
+        foreach ($this->getTranslations() as $locale => $domains) {
+            foreach ($formats as $format) {
+                $content = $this->engine->render(
+                    'BazingaJsTranslationBundle::getTranslations.' . $format . '.twig',
+                    array(
+                        'translations' => array($locale => $domains),
+                        'include_config' => false,
+                    )
+                );
+
+                $file = sprintf(
+                    '%s/%s',
+                    $target,
+                    strtr(
+                        $route->getPath(),
+                        array(
+                            '{domain}' => sprintf('%s/%s', $locale, $locale),
```

Comment:
So what do you propose? All JS and JSON files under `translations` folder?

I agree that some people might prefer grouping the files by format, but that is true not just for this case. When dumping translations per domain you could also prefer that instead of the current solution (a folder per domain). For that reason, I think that option shouldn't be considered now.